server/store/store.go

```diff
@@ -1109,11 +1113,20 @@ func (s *SQLStore) PrefillWhitelist() error {
 	return nil
 }
 
-func (s *SQLStore) GetSizeOfWhitelist() (int, error) {
+func (s *SQLStore) GetSizeOfWhitelist(tx *sql.Tx) (int, error) {
 	query := s.getQueryBuilder().Select("count(*)").From(whitelistedUsersTableName)
-	rows, err := query.Query()
-	if err != nil {
-		return 0, err
+	var rows *sql.Rows
+	var err error
+	if tx == nil {
+		rows, err = query.Query()
+		if err != nil {
+			return 0, err
+		}
```

Comment:
we can have these outside conditions as this is repeated for the below condition as well